share_light.module

```diff
@@ -129,15 +130,22 @@ function share_light_token_info() {
  */
 function share_light_tokens($type, $tokens, array $data = array(), array $options = array()) {
   $replacements = array();
-  if ($type == 'share' && isset($data['share'])) {
+  if ($type == 'share_email' && isset($data['share_email'])) {
```

Comment:
Renaming the tokens shouldn’t be part of a coding-style fixup commit. Also this needs migrations for existing databases.